### PR TITLE
Annotate methods which return explicit null

### PR DESCRIPTION
**Pull request by oleg-nenashev**
There is a risk of NPEs in some cases, e.g. in "*Strategy#getPriority()" API. No related JIRA issues, but I would rather fix it.

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
I forgot about @reviewbybees

File: src/main/java/jenkins/advancedqueue/JobGroup.java [hunk 1]
```diff
@@ -139,6 +140,7 @@ public int getPriority() {
 	 * @deprecated Used in 2.x now replaced with dynamic {@link JobGroup#jobGroupStrategy}, will return the view
 	 */
 	@Deprecated
+        @CheckForNull
```
Comment (abayer) [inline src/main/java/jenkins/advancedqueue/JobGroup.java]: Indent. =)

File: src/main/java/jenkins/advancedqueue/PriorityConfiguration.java [hunk 2]
```diff
@@ -236,7 +238,8 @@ private PriorityConfigurationCallback getPriorityInternal(Queue.Item item, Prior
 		return priorityCallback.setPrioritySelection(PrioritySorterConfiguration.get().getStrategy().getDefaultPriority());
 	}
 
-	public JobGroup getJobGroup(PriorityConfigurationCallback priorityCallback, Job<?, ?> job) {
+        @CheckForNull
```
Comment (abayer) [inline src/main/java/jenkins/advancedqueue/PriorityConfiguration.java]: Indent again?

**Comment by oleg-nenashev** [comment]
@abayer As a plugin maintainer I have officially gave up with maintaining tab indenting in it. Once I have zero pending PRs, I will just refactor all the code

**Pull request merged**